### PR TITLE
update the demo with the bullet_utils changes

### PR DESCRIPTION
**Pull request by MaximilienNaveau**
[//]: # "Thanks for your contribution.  To make life of the reviewers easier,"
[//]: # "please give this pull request a meaningful title and provide the"
[//]: # "requested information below."

## Description

[//]: # "Description of what you did.  If it is more complex, consider to add a"
[//]: # "'Summary' section."

Update the demos to take into account the PR in bullet_utils

## How I Tested

[//]: # "Explain how you tested your changes"

Ran all demos on my laptop.

## Do not merge before

[//]: # "Link other open pull request here that need to be merged first."
[//]: # "Remove this section if it does not apply."

- [ ]  https://github.com/machines-in-motion/bullet_utils/pull/7/

## I fulfilled the following requirements

[//]: # "Please make sure you followed these steps before requesting a review."
[//]: # "Check the boxes in the list below, when done."

- [ ] All new code is formatted according to our style guide (for C++ run clang-format, for Python, run flake8 and fix all warnings).
- [ ] All new functions/classes are documented and existing documentation is updated according to changes.
- [ ] No commented code from testing/debugging is kept (unless there is a good reason to keep it).

### CODE REVIEW

File: demos/demo_simulate_solo12.py [hunk 1]
```diff
@@ -17,11 +17,17 @@
 
 if __name__ == "__main__":
 
-    # Create a Pybullet simulation environment
+# ! Create a Pybullet simulation environment before any robots !
```
Comment (jviereck) [inline demos/demo_simulate_solo12.py]: The indention is off here.

File: demos/demo_simulate_solo8.py [hunk 2]
```diff
@@ -17,11 +17,17 @@
 
 if __name__ == "__main__":
 
-    # Create a Pybullet simulation environment
+# ! Create a Pybullet simulation environment before any robots !
```
Comment (jviereck) [inline demos/demo_simulate_solo8.py]: Indention is off here.

File: demos/demo_simulate_solo12.py [hunk 3]
```diff
@@ -17,11 +17,17 @@
...
     env = BulletEnvWithGround()
 
-    # Create a robot instance. This initializes the simulator as well.
-    robot = env.add_robot(Solo12Robot)
+    # Create a robot instance. This adds the robot to the simulator as well.
+    robot = Solo12Robot(useFixedBase=False)
```
Comment (jviereck) [inline demos/demo_simulate_solo12.py]: The `useFixedBase=False` is the default, no? So maybe we can just remove it to make the constructor shorter.

File: demos/demo_simulate_solo8.py [hunk 4]
```diff
@@ -17,11 +17,17 @@
...
     env = BulletEnvWithGround()
 
-    # Create a robot instance. This initializes the simulator as well.
-    robot = env.add_robot(Solo8Robot)
+    # Create a robot instance. This adds the robot to the simulator as well.
+    robot = Solo8Robot(useFixedBase=False)
```
Comment (jviereck) [inline demos/demo_simulate_solo8.py]: Similar as above, I think the `useFixedBase = False` is the default?

**Review by jviereck — APPROVED** [review]
Small comments but otherwise looks good.

File: demos/demo_simulate_solo8.py (hunk 4 above)
Comment (MaximilienNaveau) [reply demos/demo_simulate_solo8.py]: ok yeah, makes sense

**Pull request merged**